sass/_colors.scss

```diff
@@ -1 +1,2 @@
-$blue: #42f4eb;
\ No newline at end of file
+$blue: #c2fcf6;
```

Comment:
be consistent with spacing.  make sure there are the same number of enters after each class.  no empty lines before closing curly brackets etc